packages/gatsby-plugin-manifest/README.md

```diff
@@ -26,12 +26,28 @@ plugins: [
   {
     resolve: `gatsby-plugin-manifest`,
     options: {
-      "name": "GatsbyJS",
-      "short_name": "GatsbyJS",
-      "start_url": "/",
-      "background_color": "#f7f0eb",
-      "theme_color": "#a2466c",
-      "display": "minimal-ui",
+      name: "GatsbyJS",
+      short_name: "GatsbyJS",
+      start_url: "/",
+      background_color: "#f7f0eb",
+      theme_color: "#a2466c",
+      display: "minimal-ui",
+      icons: [
+        {
+          // Everything in /src/static will be copied to an equivalent
```

Comment:
The folder is at the root of the project not inside src. https://www.gatsbyjs.org/docs/adding-images-fonts-files/#using-the-static-folder